lastminute.yml

```diff
@@ -182,7 +184,7 @@
   vars:
     judging_notes: "{{ 'files/%s.JudgingNotes.%s.pdf'|format(contest_detail, papersize) }}"
     tech_notes: "{{ 'files/%s.TechNotes.%s.pdf'|format(contest_detail, papersize) }}"
-    judging_notes_addendum: "{{ 'files/%s.JudgingNotes.Addendum.%s.pdf'|format(contest_detail, papersize) }}"
+    judging_addendum_notes: "{{ 'files/%s.JudgingNotes.Addendum.%s.pdf'|format(contest_detail, papersize) }}"
 
   tasks:
```

Comment:
this seems wrong

Reply:
![image](https://github.com/icpcsysops/ansible/assets/672279/7df53bb8-0ebd-4c18-a37c-a06f0869ef68)

Below it tris to reference a variable called `judging_addendum_notes`, and prior to this change that variable is undefined/missing.